es6-shim.js

```diff
@@ -1850,7 +1850,7 @@
       if (x === -1) { return -Infinity; }
       if (x === 1) { return Infinity; }
       if (x === 0) { return x; }
-      return 0.5 * _log((1 + x) / (1 - x));
+      return Math.sign(x) * Math.log1p(2 * Math.abs(x) / (1 - Math.abs(x))) / 2;
```

Comment:
`_abs` is already available as a cached `Math.abs` - we should cache `sign` and `log1p` as well. (i'll make a cached `Math.sign` available on master shortly, so you can just rebase on top of it and use `_sign`)